src/app/modules/shared/components/details-fields/details-fields.component.spec.ts

```diff
@@ -300,14 +302,6 @@ describe('DetailsFieldsComponent', () => {
     expect(component.goingToWorkOnThis).toBeFalse();
   });
 
-  it('when editing entry that already finished, then the entry should not be marked as running', () => {
```

Comment:
This tests seems to be useful. Why do we deleted?

Reply:
@Angeluz-07 this test are repeated.